packages/gluestick/src/config/webpack/webpack.config.client.js

```diff
@@ -52,6 +51,15 @@ module.exports = (
     };
   });
   config.plugins.push(
+    new webpack.optimize.CommonsChunkPlugin({
```

Comment:
Both dev and prod now use CommonsChunk. We'll need to tweak settings to see what we end up with on the consumer side.